pkgs/development/python-modules/pysubs2/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, pytest-cov
+, mypy
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "pysubs2";
+  version = "1.6.1";
+  format = "pyproject";
+
+  # pypi is missing the 'tests/data' directory, needed to run tests
```

Comment:
```suggestion
```

Reply:
I removed this, although I'm a bit confused why. I don't care strongly about it though.

Reply:
That is very common and we don't reason about it.